### PR TITLE
CORnetV1

### PR DESCRIPTION
**Pull request by tiagogmarques**
Added CORnetV1 models to candidate_models and wrapper for activations_model

### CODE REVIEW

File: candidate_models/base_models/__init__.py [hunk 1]
```diff
@@ -265,14 +270,15 @@ def robust_model(function, image_size):
         _logger.debug(f"Downloading weights for resnet-50-robust from {url} to {weights_path}")
         os.makedirs(weightsdir_path, exist_ok=True)
         request.urlretrieve(url, weights_path)
-    checkpoint = load(weights_path, map_location=lambda storage, location: 'cpu')
+    checkpoint = load(weights_path, map_location='cpu')
+    # checkpoint = load(weights_path)
```
Comment (mschrimpf) [inline candidate_models/base_models/__init__.py]: dead code

File: candidate_models/base_models/cornetv1/__init__.py [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+                   if coord != 'neuroid_id'},  # otherwise, neuroid dim will be as large as before with nans
+                **{'time_step': [int(timestep)]}
+            }, dims=['time_step'] + list(key_activations.dims))
+        activations = list(activations.values())
+        activations = merge_data_arrays(activations)
+        # rebuild neuroid_id without timestep
+        neuroid_id = [".".join([f"{value}" for value in values]) for values in zip(*[
+            activations[coord].values for coord in ['model', 'region', 'neuroid_num']])]
+        activations['neuroid_id'] = 'neuroid', neuroid_id
+        return activations
```
Comment (mschrimpf) [inline candidate_models/base_models/cornetv1/__init__.py]: this is a lot of duplicate code. We can merge it like this, but in order to benefit from potential updates, you might want to just import from the other cornet code

**Comment by tiagogmarques** [comment]
No, sorry. That shouldn’t be there… I had it there so that Joel could use the model. I should make a separate repo for that, then...

**Comment by mschrimpf** [comment]
repo or branch yea. I'll go ahead and close this, feel free to re-open when you're ready.

**Pull request closed without merging**